lib/AppConfig.php

```diff
@@ -20,7 +20,6 @@
 class AppConfig {
 	private $appName = 'richdocuments';
 	private $defaults = [
-		'wopi_url' => 'https://localhost:9980',
```

Comment:
I'm not sure about the consequences of this change... Taking into account that it's expected the admin to adjust the url anyway, I'd prefer to keep it.

Reply:
If this is irrelevant - I am for leaving it out.

For the appliance we need this to be empty on start to so we can set the appropriate settings for onlyoffice and collabora integration. A default value prevent those automatic settings. That's why I requested to removal of this default value.

Reply:
@jvillafanez I deep analyzed, no consequence